### PR TITLE
add circleci integration example

### PR DESCRIPTION
**Pull request by kiddo3**
### What I did
Add an example on how to integrate hadolint with CircleCI
### How I did it
Writing the doc with the current orb example
### How to verify it
-

### CODE REVIEW

File: docs/INTEGRATION.md [hunk 1]
```diff
@@ -88,6 +88,25 @@ Add the following job to your project's `.drone.yml` pipeline (drone version 1.0
...
 
+## CircleCI
+
+For CircleCI integration use the [docker orb](https://circleci.com/orbs/registry/orb/circleci/docker).
+
+Update your project's `.circleci/config.yml` pipeline (workflows version 2.1), adding the docker orb and you can call the job docker/hadolint:
+
+```yaml
+orbs:
+  docker: circleci/docker@0.5.13
```
Comment (SuperSandro2000) [inline docs/INTEGRATION.md]: Is the version number optional?

File: docs/INTEGRATION.md [hunk 2]
```diff
@@ -88,6 +88,25 @@ Add the following job to your project's `.drone.yml` pipeline (drone version 1.0
       - hadolint  Dockerfile
 ```
 
+## CircleCI
+
+For CircleCI integration use the [docker orb](https://circleci.com/orbs/registry/orb/circleci/docker).
+
+Update your project's `.circleci/config.yml` pipeline (workflows version 2.1), adding the docker orb and you can call the job docker/hadolint:
```
Comment (SuperSandro2000) [inline docs/INTEGRATION.md]: This needs some cleanup.

File: docs/INTEGRATION.md (hunk 1 above)
Comment (kiddo3) [reply docs/INTEGRATION.md]: No, for latest you can specify volatile. Also you can specify only the major number https://circleci.com/docs/2.0/using-orbs/#semantic-versioning-in-orbs

Comment (SuperSandro2000) [reply docs/INTEGRATION.md]: maybe this should be 0 or 0.5 or maybe even volatile if it does not break in the future.

Comment (kiddo3) [reply docs/INTEGRATION.md]: I've changed it to x.y.z, the way circleci write the orbs examples. it's ok or do you prefer versioning it ?

**Comment by zemanlx** [comment]
@kiddo3 thank you for your contribution! I see that linting job https://travis-ci.org/hadolint/hadolint/jobs/593887437 is failing because

```
docs/INTEGRATION.md
477  95:143  warning  Line must be at most 100 characters  maximum-line-length  remark-lint
```

Can you please fix it before we merge it?

**Comment by lorenzo** [comment]
Thanks!

**Pull request merged**